test/contracts/HCElection.aes

```diff
@@ -3,8 +3,9 @@ include "Pair.aes"
 
 contract interface MainStaking =
   entrypoint sorted_validators : () => list((address * int))
-  entrypoint lock_stake  : (int) => list((address * int))
-  entrypoint add_rewards : (int, list(address * int)) => unit
+  entrypoint is_validator : (address) => bool
+  entrypoint total_stake : () => int
+  stateful entrypoint post_elect : () => unit
```

Comment:
Of course you can't just revert the consensus contracts, but the tests will explain that 😅